infrastructure/main.tf

```diff
@@ -79,12 +79,12 @@ locals {
   aseName = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
   app_full_name = "${var.product}-${var.component}"
 
-  localCcdApi = "http://ccd-data-store-api-${var.env}.service.${local.aseName}.internal"
-  CcdApi = "${var.env == "preview" ? "http://ccd-data-store-api-aat.service.core-compute-aat.internal" : local.localCcdApi}"
-
-  localPdfService = "http://cmc-pdf-service-${var.env}.service.${local.aseName}.internal"
-  pdfService = "${var.env == "preview" ? "http://cmc-pdf-service-aat.service.core-compute-aat.internal" : local.localPdfService}"
+  local_env = "${(var.env == "preview" || var.env == "spreview") ? (var.env == "preview" ) ? "aat" : "saat" : var.env}"
+  local_ase = "${(var.env == "preview" || var.env == "spreview") ? (var.env == "preview" ) ? "core-compute-aat" : "core-compute-saat" : local.aseName}"
 
+  CcdApi = "http://ccd-data-store-api-${local.local_env}.service.${local.local_ase}.internal"
+  s2sCnpUrl = "http://rpe-service-auth-provider-${local.local_env}.service.${local.local_ase}.internal"
```

Comment:
the api's aren't compatible, you also need to update the client version

Reply:
This is being fixed by https://github.com/hmcts/tribunals-case-api/pull/145/files#diff-c197962302397baf3a4cc36463dce5eaR191